src/ripple/ledger/ReadView.h

```diff
@@ -247,10 +247,12 @@ class ReadView
     }
 
     /** Returns the close time of the previous ledger. */
-    std::uint32_t
+    TimeKeeper::time_point
     parentCloseTime() const
     {
-        return info().parentCloseTime;
+        using d = TimeKeeper::duration;
+        using tp = TimeKeeper::time_point;
+        return tp{d{info().parentCloseTime}};
```

Comment:
I see this three-line pattern several places in the change set. How about a function like: `template <class Clock, class Int> time_point_from_int(Int)`. We could also make it a static member, like `from_time_t`, but that won't work for standard clocks.

I understand that this is an intermediate step to where you want to go and the type of `info().parentCloseTime` and friends may changing in future PRs, so I'm OK if you don't want to add this function.